### ISSUE TITLE
Validate translations before deployments

### ISSUE THREAD

**Issue by embreinhardt** [usdigitalresponse/project-papua#78]
Mixed use of "Usted / su / está / es" and "Tú / tu / estas / eres" (should use first form). Occasionally missing the opening (upside down) question mark. Should just do a pass on any final translations

![Screen Shot 2020-04-21 at 9 12 13 PM](https://user-images.githubusercontent.com/2966080/79939908-21889400-8415-11ea-8fed-80eeba1f1ad2.png)

Also mixing accent in `direccion`
 
![Screen Shot 2020-04-21 at 9 14 26 PM](https://user-images.githubusercontent.com/2966080/79939905-20effd80-8415-11ea-8ca5-06680a4029e0.png)
